django/contrib/flatpages/admin.py

```diff
@@ -4,15 +4,16 @@
 from django.utils.translation import ugettext_lazy as _
 
 
+@admin.register(FlatPage)
 class FlatPageAdmin(admin.ModelAdmin):
     form = FlatpageForm
     fieldsets = (
         (None, {'fields': ('url', 'title', 'content', 'sites')}),
-        (_('Advanced options'), {'classes': ('collapse',),
-        'fields': ('enable_comments', 'registration_required', 'template_name')}),
+        (_('Advanced options'), {
+            'classes': ('collapse', ),
+            'fields': ('registration_required', 'template_name')
```

Comment:
We like to include trailing comma on the last item in a dict so if more items are added later, we don't have to modify this line again.